ee/tabby-ui/app/chat/page.tsx

```diff
@@ -136,6 +136,23 @@ export default function ChatPage() {
         document.execCommand('paste')
       } else if ((event.ctrlKey || event.metaKey) && event.code === 'KeyA') {
         document.execCommand('selectAll')
+      } else if (
+        event.key.toLowerCase() === 'l' &&
+        (event.ctrlKey === true || event.metaKey === true)
```

Comment:
need a way to pass the shortcut from vscode to chat-ui, i recommend add shortcut as part of InitRequest.keyboard_shortcut_options: https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/83b6748b768c9061f6b17d9fc94f7a8e9194f22d/clients/tabby-chat-panel/src/index.ts#L22